RadarSDK.xcodeproj/project.pbxproj

```diff
@@ -956,6 +959,8 @@
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				"PROVISIONING_PROFILE_SPECIFIER[sdk=macosx*]" = "";
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 6.0;
```

Comment:
I swear swift 6.0 didn't work for me with CI, it was complaining swift 6 is not available.